Description: Correctly set usec field of daedline instead of sec
 Found by Julien Cristau when reviewing
 01-use-poll-instead-of-select.patch which was based on functionality that
 went into 0.8.13. The deadline calculation incorrectly saves the
 calculated micosecondsecond value in the second value possibly causing a
 very long delay. The long delay is avoided by other sanity checking code
 in place.
Author: Julien Cristau <jcristau@debian.org>
Origin: other, http://bugs.debian.org/691640
Forwarded: not-needed
Applied-Upstream: commit, http://arthurdejong.org/viewvc/nss-pam-ldapd?revision=1850&view=revision
Reviewed-by: Arthur de Jong <arthur@arthurdejong.org>

--- a/common/tio.c
+++ b/common/tio.c
@@ -86,7 +86,7 @@ static inline void tio_get_deadline(stru
     return;
   }
   deadline->tv_sec+=timeout/1000;
-  deadline->tv_sec+=(timeout%1000)*1000;
+  deadline->tv_usec+=(timeout%1000)*1000;
 }
 
 /* update the timeout to the value that is remaining before deadline
